### ISSUE TITLE
Exploring alternatives to PZCT-02

### ISSUE THREAD

**Issue by jalakamkiranadspl** [bvarner/PZCT-02-BurdenShift#2]
Hey @bvarner, Thanks for the design. It shares good knowledge. I am trying to find an alternative current transformer for the PZCT-02 which is rather very expensive. Initially I found the turns ratio to be 1000:1 with capability of measuring 100A. Did you come across any of them? I know this is completely unrelated to your repository but I thought you might shed some light on this since I am trying make the circuit out of it.

**Comment by bvarner** [comment]
Yes, at $7 / per unit the PZCT-02 is a bit of a high price point compared to smaller diameter transformers.

Searching aliexpress for 1000:1 current transformers I found quite a few there.  IIRC you also need to account for the impedance of the transformer when sizing burden resistors. Good luck!

I chose the PZCT-02 as it was the cheapest thing I could find that would handle 200A.

**Comment by norah555** [comment]
salut j'ai un capteur PZCT-02 mais je n'arrive pas à le relier à la carte ads puis au rasberry. Il ne faut pas rajouter de résistance entre le capteur et la carte ADS ?
